api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -51,7 +55,15 @@ message UdpProxyConfig {
     option (validate.required) = true;
 
     // The upstream cluster to connect to.
```

Comment:
Please update the docs here to mention this is deprecated and to use `matcher` instead.

Reply:
Ok, done.